python/ray/_private/worker.py

```diff
@@ -1306,6 +1306,8 @@ def init(
             instance.
             4. If the provided address is "local", start a new local Ray
             instance, even if there is already an existing local Ray instance.
+            Running multiple local Ray instances is not supported and may have
+            undefined behavior.
```

Comment:
Why is it not supposed? I think this is supported.

Reply:
Should this be "Running multiple local Ray instances is not recommended and may have
undefined behavior." instead?

Reply:
You can do it as long as you provide unique ports for each Ray instance.

Reply:
@jjyao

1. Do you have an example Python snippet showing how to do this with `ray.init()`?
2. I understand it's possible, but I'm asking if it's not recommended?

Reply:
@jjyao just bumping again on the above questions